src/routes/_database/databaseApis.js

```diff
@@ -6,6 +6,6 @@ export * from './timelines/pagination'
 export * from './timelines/getStatusOrNotification'
 export * from './timelines/updateStatus'
 export * from './timelines/deletion'
-export * from './timelines/insertion'
+export { insertTimelineItems, insertStatus } from './timelines/insertion'
```

Comment:
this thing was exporting too many APIs, accidental mistake although not a big deal